packages/components/addon/components/hds/side-nav/index.hbs

```diff
@@ -0,0 +1,39 @@
+<div class={{this.classNames}} ...attributes>
+  <div class="hds-side-nav__header">
+    {{yield to="header"}}
+  </div>
+
+  <div class="hds-side-nav__body">
+    {{yield to="body"}}
+  </div>
+
+  <div class="hds-side-nav__footer">
+    {{yield to="footer"}}
+  </div>
+</div>
+
+{{!
+  ROUGH PLANNING NOTES (these will be deleted, for WIP planning ONLY)
+
+  Component Structure:
+  ----------------------------
+
+  * SideNav (parent wrapper component)
+    * Header (yielded wrapper component)
+      - Logo / Home Button (child) -- we should probably provide
+      - Help Dropdown (child) -- should we provide? Perhaps allow users just to add custom links?
+      - User Dropdown (child) -- should we provide? Seems like we should
+
+      - Hamburger Button / Close Button -- Need to know more about our planned design
+    * Body (yielded wrapper component)
+      - NavGroup (1 or more, wrapper/parent component)
```

Comment:
I'll probably rename these "NavGroup" child components because the name is a bit redundant.